src/index.html

```diff
@@ -17,6 +17,78 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="index.html"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol if they are meant to be page anchors. If this is meant to be a link to the homepage, consider using `/` for clarity.